roles/splunk_cluster_master/tasks/generate_ess_bundle.yml

```diff
@@ -37,8 +35,6 @@
   unarchive:
     src: "{{ ta_location.stdout_lines[0] }}"
     dest: "{{ splunk.app_paths.idxc }}"
```

Comment:
Will this also need the become/become_user since it's extracting files directly into some path within SPLUNK_HOME?

Reply:
yes I was container focused where we don't support ES/ITSI yet made adjustment

Reply:
Was there a workaround added for this fix? I'll kick off a windows ec2 test to verify as well.